jax/interpreters/partial_eval.py

```diff
@@ -1240,9 +1240,44 @@ def call_partial_eval_custom_rule(
   new_inst = [x for x, inst in zip(eqn.invars, inst_in)
               if type(x) is Var and not inst]
   return eqn_known, eqn_staged, unks_out, inst_out, new_inst + residuals
+
+def closed_call_partial_eval_custom_rule(
```

Comment:
Can we dedup this with the other call rules (e.g. by including it in the call rule 'template' `call_partial_eval_custom_rule`)? Not necessary to do now but maybe we can leave a TODO to try to deduplicate a bit.